staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -286,8 +286,8 @@ const (
 )
 
 // OwnerReference contains enough information to let you identify an owning
-// object. Currently, an owning object must be in the same namespace, so there
-// is no namespace field.
+// object. An owning object must either be in the same namespace as the
+// dependent is, or be cluster-scoped, so there is no namespace field.
```

Comment:
@caesarxuchao do u mean that a namespaced object could have a cluster-scoped owner? if so, it will break some existing logic. 

[https://github.com/kubernetes/kubernetes/blob/b7b0aae4358065b98a3db04411bec65a11eb166e/plugin/pkg/admission/gc/gc_admission.go#L192-L193](https://github.com/kubernetes/kubernetes/blob/b7b0aae4358065b98a3db04411bec65a11eb166e/plugin/pkg/admission/gc/gc_admission.go#L192-L193)

Reply:
Good catch, sent https://github.com/kubernetes/kubernetes/pull/70389.

Reply:
might read slightly better as "must ~~either~~ be in the same namespace as the dependent ~~is~~, or be cluster-scoped, so there is no namespace field."

also needs to regenerate to update docs, since this is an API type